src/main/check.cpp

```diff
@@ -14,7 +14,11 @@
 #include "asm_unmarshal.hpp"
 #include "config.hpp"
 #include "crab_verifier.hpp"
-#include "memsize.hpp"
+#ifdef _WIN32
+#include "memsize_windows.hpp"
+#else
+#include "memsize_linux.hpp"
+#endif
```

Comment:
Maybe we should have `memsize.hpp` and `memsize_<os>.cpp` so we won't include `windows.h` / `unistd.h` every time? We can call it `os_utils.cpp` and also put `systemTime()` there.

Reply:
What do you mean by "every time"?  check.cpp is the only file that includes memsize.

Reply:
Okay.